twingate/resource_connector_test.go

```diff
@@ -21,25 +23,27 @@ func TestAccRemoteConnector_basic(t *testing.T) {
 			CheckDestroy:      testAccCheckTwingateConnectorDestroy,
 			Steps: []resource.TestStep{
 				{
-					Config: testTwingateConnector(remoteNetworkName),
+					Config: testTwingateConnectorWithCustomName(remoteNetworkName, connectorName),
 					Check: resource.ComposeTestCheckFunc(
 						testAccCheckTwingateConnectorExists(connectorResource, remoteNetworkResource),
+						resource.TestMatchResourceAttr(connectorResource, "name", regexp.MustCompile("tf-acc.*")),
 					),
 				},
 			},
 		})
 	})
 }
 
-func testTwingateConnector(remoteNetworkName string) string {
+func testTwingateConnectorWithCustomName(remoteNetworkName string, connectorName string) string {
```

Comment:
let's have two test one for connector with name and one for connector without name